### PR TITLE
Update index

### PR DESCRIPTION
**Pull request by jospoortvliet**
* updated metadata
* update piwik & mautic code
* update footer

Sorry, I should've noted I was still working on these...

### CODE REVIEW

**Comment by jospoortvliet** [comment]
nextcloud_documentation_-_2018-03-15_09 33 48

**Comment by jospoortvliet** [comment]
ready to go...

File: index.html [hunk 1]
```diff
@@ -2,16 +2,45 @@
...
+	<!-- Open Graph data -->
+	<meta property="og:title" content="Nextcloud Documentation" />
+	<meta property="og:type" content="article" />
+	<meta property="og:url" content="http://nextcloud.com" />
+	<meta property="og:description" content="Nextcloud is an open source, self-hosted file share and communication platform. Access  &amp; sync your files, contacts, calendars  &amp; communicate and collaborate across your devices. You decide what happens with your data, where it is and who can access it!" />
+	<meta property="og:site_name" content="Nextcloud" />
+	<meta property="fb:admins" content="1032807203462807" />
+
+	<!-- set images for generic, twitter and Open Graph -->
+	<meta itemprop="image" content="https://docs.nextcloud.com/server/13/user_manual/_images/files_page.png">
```
Comment (MorrisJobke) [inline index.html]: Do you really want to have this in the preview?

files_page

Comment (jospoortvliet) [reply index.html]: yes, actually - this is our documentation site and this was the most obvious image that makes that clear. ;-)

Comment (jospoortvliet) [reply index.html]: I spent plenty time finding it :D

**Comment by jospoortvliet** [comment]
Any other input? If not, I'd love to see this go live too :100:

**Comment by MorrisJobke** [comment]
> Any other input? also from @splitt3r or @MariusBluem or others - if not, I'd love to see this go live too 💯

Every day at 3 pm ;)

File: index.html [hunk 2]
```diff
@@ -2,16 +2,45 @@
...
+	<meta name="twitter:card" content="summary_large_image">
+	<meta name="twitter:site" content="@nextclouders">
+	<meta name="twitter:url" content="https://docs.nextcloud.com">
+	<meta name="twitter:title" content="Nextcloud Documentation">
+	<meta name="twitter:description" content="Nextcloud is an open source, self-hosted file share and communication platform. Access  &amp; sync your files, contacts, calendars  &amp; communicate and collaborate across your devices.">
+
+	<!-- Open Graph data -->
+	<meta property="og:title" content="Nextcloud Documentation" />
+	<meta property="og:type" content="article" />
+	<meta property="og:url" content="http://nextcloud.com" />
```
Comment (splitt3r) [inline index.html]: For Twitter it is docs.nextcloud.com here it is only nextcloud.com?

File: index.html [hunk 3]
```diff
@@ -210,34 +244,65 @@ <h2>Reporting Documentation Bugs &amp; How to Contribute<a class="headerlink" hr
...
+					<li><a class="reference external" href="https://nextcloud.com/community/code-of-conduct/">Code of Conduct</a></li>
+					<li><a class="reference external" href="https://nextcloud.com/privacy">Privacy</a></li>
+					<li><a class="reference external" href="https://nextcloud.com/impressum">Legal notice</a></li>
+				</ul>
+			</div>
+		</div>
+		<div class="col-md-2 col-md-offset-1 col-sm-3">
+			<div class="footer-nav">
+				<h4 >Resources</h4>
+				<ul >
```
Comment (splitt3r) [inline index.html]: Spaces 😁

Comment (jospoortvliet) [reply index.html]: Yeah, I know - lazy, I didn't remove them...

File: index.html (hunk 2 above)
Comment (jospoortvliet) [reply index.html]: Good catch, not urgent - but - could you make a separate PR to fix it?

**Comment by jospoortvliet** [comment]
Thanks for spending the time dude!

**Pull request merged**